Устройство состоит из трех деталей. Для первой детали вероятность выйти из строя в первый месяц равна 0.1, для второй - 0.2, для третьей - 0.25. 
Какова вероятность того, что в первый месяц выйдут из строя: а). все детали б). только две детали в). 
хотя бы одна деталь г). от одной до двух деталей?

In [9]:
p_1 = 0.1
p_2 = 0.2
p_3 = 0.25
q_1 = 0.9
q_2 = 0.8
q_3 = 0.75
X = q_1*q_2 * q_3 # вероятность того, что в первый месяц выйдут из строя ни одна деталь 
P_c = p_1*p_2*p_3
P_c
print("Вероятность того, что в первый месяц выйдут из строя все детали= "+str(P_c*100) +" %")

Вероятность того, что в первый месяц выйдут из строя все детали= 0.5000000000000001 %


In [14]:
P_B1 = (p_1*p_2*q_3)+(p_1 *q_2*p_3)+(q_1*p_2*p_3)

print("Вероятность того, что в первый месяц выйдут из строя только две детали = "+str(P_B1*100) +" %")

Вероятность того, что в первый месяц выйдут из строя только две детали = 0.08000000000000002 %


In [20]:
P_B2 = 1-X # Хотя бы одна деталь выйдет из строя 
P_B3 = (p_1*q_2*q_3)+(q_1 *p_2*q_3)+(q_1*q_2*p_3) # Только одна деталь выйдет из строя
P_B4 = P_B1+P_B3 # Хотя бы одна деталь выйдет из строя 
P_B2 ,P_B4 # От одной до двух деталей выйдет из строя 

print("Вероятность того, что в первый месяц выйдут из строя от одной до двух деталей = "+str(P_B4*100) +" %")
print("Вероятность того, что в первый месяц выйдут из строя хотя бы одна деталь = "+str(P_B2*100) +" %")

Вероятность того, что в первый месяц выйдут из строя от одной до двух деталей = 45.5 %
Вероятность того, что в первый месяц выйдут из строя хотя бы одна деталь = 46.0 %


а) Вероятность того, что в первый месяц выйдут из строя все детали = 0.5000000000000001 %
б) Вероятность того, что в первый месяц выйдут из строя только две детали = 0.08000000000000002 %
в) Вероятность того, что в первый месяц выйдут из строя от одной до двух деталей = 45.5 %
г) Вероятность того, что в первый месяц выйдут из строя хотя бы одна деталь = 46.0 %